management/src/main/java/org/kaazing/gateway/management/monitoring/entity/manager/impl/ServiceSessionCounterManagerImpl.java

```diff
@@ -29,6 +29,7 @@
 public class ServiceSessionCounterManagerImpl implements
```

Comment:
Please rename this class to ServiceCounters so it can be used for all service-level counters as we go forward (it's worth looking at Aeron class [SystemCounters](https://github.com/real-logic/Aeron/blob/master/aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/SystemCounters.java) which is similar - maybe you already saw it). A few easy changes will be enough to make it more general (rename incrementCounters to incrementSessionCounters and rename decrementCounters to decrementSessionCounters). I would also recommend simplifying the getXxxCounter methods by renaming them simply to xxxCounter(), following the pattern in SystemCounters, for example numberOfSessionsCounter instead of getNumberOfSessionsCounter, to make calling code more concise and simpler to read.